.github/workflows/_rocm-test.yml

```diff
@@ -61,6 +65,17 @@ jobs:
       - name: Setup ROCm
         uses: ./.github/actions/setup-rocm
 
+      - name: configure aws credentials
+        id: aws_creds
+        uses: aws-actions/configure-aws-credentials@v1.7.0
+        with:
+          role-to-assume: arn:aws:iam::308535385114:role/gha_workflow_pytorch_artifacts
```

Comment:
Please mention in PR description, what privileges this role gives to the runner?  Right now I see it gives one ability to ecr:PutImage, which is not necessary to run tests, is it?

Reply:
Thank you. Adjusted this role to allow read only ecr actions.